.python-version

```diff
@@ -1 +1,2 @@
-2.7.14
+2.7.6
```

Comment:
All versions of Python < 2.7.9 have a known security flaw that should be avoided.  Now that there is guaranteed to _never_ be another release of legacy Python, I see no reason not to pin to [Python 2.7.18](https://www.python.org/downloads/release/python-2718/).